services/initChatAnonymous/initChatAnonymousServices.ts

```diff
@@ -52,7 +52,7 @@ type InitChatAnonymousPayload = {
   anon_user_info_color_name: string;
   anon_user_info_emoji_code: string;
   anon_user_info_emoji_name: string;
-  members: string[];
+  member: string[];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `members` to `member` might cause issues if the rest of your codebase is expecting an array of members. If you're sure that only one member will be present, this change is fine. However, if there could be more than one member, it would be better to keep the property as an array and rename it to `members` for clarity.
```diff
-   member: string[];
+   members: string[];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->